### ISSUE TITLE
Show full hook ID

### ISSUE THREAD

**Issue by klahnakoski** [taskcluster/taskcluster#2835]
In an effort to update [configuration for a project](https://github.com/mozilla/community-tc-config/blob/master/config/projects/cia.yml), I would like to know the hook ID. I can navigate to the hook I am interested in, but hook ID shown seems incomplete.  What is the full hook ID?

![image](https://user-images.githubusercontent.com/2334429/81986329-98c0da80-9605-11ea-9eb4-387a9396ae5a.png)

Right now I am guessing it is `hook-id:project-cia/etl-schedulers`, but only because I was lucky to have a hook ID prefix in the configuration file already.  I will not be so lucky next time.

**Comment by helfi92** [comment]
As far as I understand the hook ID is simply `etl-schedulers` as displayed in the UI. @taskcluster/services-reviewers can you confirm?

**Comment by djmitche** [comment]
that's correct -- in that view, hookGroupId is project-cia nad hookId is etl-schedulers.

**Comment by djmitche** [comment]
Ah, I think you're interested in the name of the role.  It might be useful to add a role expansion to the view.  Something like

> This hook will be created using the scope `assume:hook-id:<hookGroupId>/<hookId>`.  That scope expands to ...

**Comment by klahnakoski** [comment]
@djmitche   

I was specifically referring to `hook-id:project-cia/*`, which I thought was a hook, but maybe is a role; you know better.

I am still confused about what the ID (?GUID?)  prefixes are;  I had thought `hook-id:`, `assume:` or 
`auth:aws-s3:read-write:` was a datatype or directive outside user control.  But it seems I had given my role a `hook-id:` prefix?

**Comment by djmitche** [comment]
The strings you're seeing are [scopes](https://docs.taskcluster.net/docs/manual/design/apis/hawk/scopes).  The link between a hook and the scopes it uses to create tasks is [here](https://docs.taskcluster.net/docs/reference/core/hooks/#hooks).

**Comment by klahnakoski** [comment]
OK, I must give that some more thought.  Thank you.

**Comment by djmitche** [comment]
https://github.com/taskcluster/taskcluster/issues/2835#issuecomment-628903215 is still probably worth doing.  Even without the expansion, it would be useful.
